app/qml/form/MMFormPage.qml

```diff
@@ -281,7 +281,11 @@ Page {
         property var fieldWidget: model.EditorWidget
         property var fieldConfig: model.EditorWidgetConfig
 
-        property bool fieldIsReadOnly: root.state === "readOnly" || !AttributeEditable
+        // Following issue #3457, error message can be displayed even for non-editable attributes
+        // Thus, we split attributes indicating edit mode for formPage and attribute editability
```

Comment:
I don't think we need this comment when we rename the properties appropriately :)